lisa/base_tools/sed.py

```diff
@@ -10,13 +10,19 @@ def command(self) -> str:
         return "sed"
 
     def replace(
-        self, searched: str, replaced: str, file: str, sudo: bool = False
+        self, searched: str, original: str, replaced: str, file: str, sudo: bool = False
```

Comment:
So the first parameter should be called as `match_lines` like manual. It should have default value `""`, because it's optional. And other parameters should be called `regexp` and `replacement`.

refer to https://www.gnu.org/software/sed/manual/sed.html#Regexp-Addresses

Reply:
It's not fully fixed.

Reply:
Have changed as your comments.